File: backend/src/main/java/cm/mvtech/_minexpo/model/dto/ProjectSessionResponse.java
package cm.mvtech._minexpo.model.dto;

import cm.mvtech._minexpo.enums.ProjectStatus;

import java.time.LocalDateTime;
import java.util.UUID;

public record ProjectSessionResponse(
        UUID id,
        String theme,
        String subject,
        String description,
        String academicLevel,
        String language,
        ProjectStatus projectStatus,
        int expectedPages,
        LocalDateTime createdAt,
        LocalDateTime updatedAt
) {
}

File: frontend/src/app/models/ProjectSessionResponse.ts
import { ProjectStatus } from "../enum/ProjectStatus";

export interface ProjectSessionResponse {
    id: string;
    theme: string;
    subject: string;
    description: string | undefined;
    academicLevel: string;
    language: string;
    projectStatus: ProjectStatus;
    expectedPages: number | undefined;
    createdAt: Date | undefined;
    updatedAt: Date | undefined;
}